graphql/util.go

```diff
@@ -1290,3 +1290,63 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+func GroupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*GroupedProjects, error) {
+	groupsMap := make(map[string][]*restModel.APIProjectRef)
+
+	for _, p := range projects {
+		var groupName string
+		groupName = fmt.Sprintf("%s/%s", p.Owner, p.Repo)
```

Comment:
can we do groupName := instead of declaring the var

Reply:
oops, leftover from before the refactor. thanks!